build-system/tasks/helpers.js

```diff
@@ -560,6 +560,7 @@ async function compileJsWithEsbuild(srcDir, srcFilename, destDir, options) {
       outfile: destFile,
       plugins: [plugin],
       minify: options.minify,
```

Comment:
You'll get much better file sizes using `terser` on the output. I saved 1kb brotli compressed on `smartadserver.js`.

Reply:
Makes sense to apply this recommendation.

Reply:
Added teaser and it indeed cut 3kb before brotli.